pom.xml

```diff
@@ -1,15 +1,18 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?><project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
 		<groupId>org.jenkins-ci.plugins</groupId>
 		<artifactId>plugin</artifactId>
-		<version>3.43</version>
+		<version>4.7</version>
 		<relativePath />
 	</parent>
 
 	<properties>
-		<jenkins.version>2.59</jenkins.version>
-                <java.level>8</java.level>
+		<jenkins.version>2.249.1</jenkins.version>
+		<java.level>8</java.level>
+		<hpi.compatibleSinceVersion>2.249.1</hpi.compatibleSinceVersion>
```

Comment:
It's unfortunate that this line made it in here, as this is actually used to mark that an incompatible data change has been made in the plugin and so the configuration that has been saved by previous versions of the plugin may not be able to be brought forward into this version (i.e., the version we're compatible with since is actually our _own_ version and not the Jenkins version; that's specified in the `jenkins.version` field). See https://wiki.jenkins.io/display/JENKINS/Marking+a+new+plugin+version+as+incompatible+with+older+versions for more details.

Reply:
PR #26 should resolve this.